### PR TITLE
Hotfix/links

### PR DESCRIPTION
**Pull request by BorjaOuterelo**
Fix all CI link issues.

### CODE REVIEW

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ URL: http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8376277&isnumber=83
 
 * Ralph Lange: Callback-group-level Executor for ROS 2. Lightning talk at ROSCon 2018. Madrid, Spain. Sep 2018. [[Slides]](https://roscon.ros.org/2018/presentations/ROSCon2018_Lightning1_4.pdf) [[Video]](https://vimeo.com/292707644)
 
-* [CB2019]<a name="CB2019"> </a> D. Casini, T. Blaß, I. Lütkebohle, B. Brandenburg: Response-Time Analysis of ROS 2 Processing Chains under Reservation-Based Scheduling, in Euromicro-Conference on Real-Time Systems 2019. [[Paper](http://drops.dagstuhl.de/opus/volltexte/2019/10743/)].[[slides]](https://www.ecrts.org/wp-content/uploads/2019/07/casini.pdf)
+* [CB2019]<a name="CB2019"> </a> D. Casini, T. Blaß, I. Lütkebohle, B. Brandenburg: Response-Time Analysis of ROS 2 Processing Chains under Reservation-Based Scheduling, in Euromicro-Conference on Real-Time Systems 2019. [[Paper](http://drops.dagstuhl.de/opus/volltexte/2019/10743/)].[[slides]](https://t-blass.de/talks/ECRTS2019.pdf)
```
Comment (iluetkeb) [inline _docs/concepts/client_library/real-time_executor/index.md]:
```suggestion
* [CB2019]<a name="CB2019"> </a> D. Casini, T. Blaß, I. Lütkebohle, B. Brandenburg: Response-Time Analysis of ROS 2 Processing Chains under Reservation-Based Scheduling, in Euromicro-Conference on Real-Time Systems 2019. [[Paper](http://drops.dagstuhl.de/opus/volltexte/2019/10743/)].[[slides]](http://drops.dagstuhl.de/opus/volltexte/2019/10743/pdf/LIPIcs-ECRTS-2019-6.pdf)
```

File: _docs/tutorials/advanced/6lowpan_nuttx_rpi/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This guide will show how to set-up a Raspberry Pi 3 (RPI) running Raspbian and a
 - Olimex-STM32-E407 board.
 - Two PMODRF2 module which are base on the MRF24J40 module.
 - PC with Ubuntu (It works fine with Ubuntu 16.04)
-- NuttX source code, you can find a Docker file with all the tools [here](https://github.com/micro-ROS/docker/tree/master/Embedded/NuttX/development/stm32-e407).
+- NuttX source code, you can find a Docker file with all the tools [here](https://github.com/micro-ROS/docker/tree/11622d6e0602cd03c59b730097d15ab6f1061d73/Embedded/NuttX/development/stm32-e407).
```
Comment (iluetkeb) [inline _docs/tutorials/advanced/6lowpan_nuttx_rpi/index.md]: Do we really want to point to a specific commit?

File: _docs/tutorials/basic/getting_started_embedded/index.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ For that purpose, we are going to use a [Olimex STM32-E407](https://www.olimex.c
 
 At this example, we are going to flash the board with a binary that contains a NuttX App. This NuttX App is a micro-ROS client. This client will try to connect to the micro-ROS Agent running at your computer and exchange data using Micro XRCE-DDS.
 
-For that purpose, firs install [this](https://github.com/micro-ROS/docker/tree/master/Embedded/NuttX/precompiled/micro-XRCE-DDS/olimex-stm32-e407) container at your Linux machine.
+For that purpose, firs install [this](https://github.com/micro-ROS/docker/tree/11622d6e0602cd03c59b730097d15ab6f1061d73/Embedded/NuttX/precompiled/micro-ROS/olimex_stm32-e407) container at your Linux machine.
```
Comment (iluetkeb) [inline _docs/tutorials/basic/getting_started_embedded/index.md]: same as above

File: _docs/tutorials/basic/getting_started_embedded/index.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ _Need to complete the tutorial!_
 
 In this case, instead of running the complete micro-ROS stuck, we will run a DDS level example. For that purpose, the embedded board will execute a Micro XRCE-DDS example that will communicate with an Agent that is running in your Linux machine.
 
-First of all, downlod the [Docker machine](https://github.com/micro-ROS/docker/blob/master/Embedded/NuttX/precompiled/micro-ROS/olimex_stm32-e407/Dockerfile) containing all the tools. Along with all the dependencies, this image contains micro-ROS publish/subscribe pre-built examples ready to flash and run on the board.
+First of all, downlod the [Docker machine](https://github.com/micro-ROS/docker/blob/11622d6e0602cd03c59b730097d15ab6f1061d73/Embedded/NuttX/precompiled/micro-ROS/olimex_stm32-e407/Dockerfile) containing all the tools. Along with all the dependencies, this image contains micro-ROS publish/subscribe pre-built examples ready to flash and run on the board.
```
Comment (iluetkeb) [inline _docs/tutorials/basic/getting_started_embedded/index.md]: and here

File: _docs/concepts/client_library/real-time_executor/index.md (hunk 1 above)
Comment (BorjaOuterelo) [reply _docs/concepts/client_library/real-time_executor/index.md]: I think we had two pdf here, the paper and the slides, right? The slides host is not valid anymore, probably getting ready for 2020 conference.
Your suggestion is the paper right?

File: _docs/tutorials/advanced/6lowpan_nuttx_rpi/index.md (hunk 2 above)
Comment (BorjaOuterelo) [reply _docs/tutorials/advanced/6lowpan_nuttx_rpi/index.md]: Solved

File: _docs/concepts/client_library/real-time_executor/index.md (hunk 1 above)
Comment (iluetkeb) [reply _docs/concepts/client_library/real-time_executor/index.md]: Ah! Yes, you are correct, my link is the paper only.

**Pull request merged**